### DISCUSSION TITLE
Can ArchiveBox serve a complete local version of a site which redirect links on it to their local counterparts?

### DISCUSSION THREAD

**Discussion by xusiai** [ArchiveBox/ArchiveBox#1329] [Q&A] [answered]
Can ArchiveBox create an archived version of a site - including same domain links - and serve it as a local version where links redirect to their local counterparts e.g. clicking an _about_ link on localdomain/index.html leads to localdomain/about.html rather than originaldomain/about.html

I have looked around the wiki for an answer and spun up a docker container to test ArchiveBox - including running the archiving process with depth=1 - but haven't been able to achieve this yet so I'm asking here if this is within the scope of ArchiveBox. If the answer is documented in the wiki and I have missed it I apologize.

**Comment by pirate** [comment, marked as answer]
ArchiveBox is not primarily designed for this use-case. You'd be better off using browsertrix-crawler, SiteSucker, or another solution from our alternatives list: https://github.com/ArchiveBox/ArchiveBox/wiki/Web-Archiving-Community#other-archivebox-alternatives

More info here: https://github.com/ArchiveBox/ArchiveBox/issues/191
